docs/reference/release-notes/highlights-6.5.0.asciidoc

```diff
@@ -15,7 +15,9 @@ a `<clustername>_audit.log` file in the logs directory. This file is new to
 version 6.5 and prints audit entries as JSON documents. 
 
 For backwards compatibility purposes, a `<clustername>_access.log` with the old style of 
-formatting is also generated. See also <<breaking_65_settings_changes>>. 
+formatting is also generated. See also <<breaking_65_settings_changes>>. Due to this,
+the logging throughput effectively doubles. It is recommended to enable only the
+new audit format in production (adjust `log4j2.properties`).
```

Comment:
> (adjust `log4j2.properties`)

Is this meant to imply that it's advised to adjust the settings in that file instead of via the API?

Reply:
I would say so, yes. The reason being that there are multiple settings they might wish to tweak, which are reachable form the config file but not the API. And people might have upgrade procedures that overwrite this file, so it's better to mention the file anyway.
But as with any logger, the guidelines are as always: https://www.elastic.co/guide/en/elasticsearch/reference/current/logging.html